README.md

```diff
@@ -26,8 +26,6 @@ This branch works with Xcode 10.0 and supports iOS 11.0+.
 
 This branch is written in Swift 4.2.
```

Comment:
You should clarify that PRs should be submitted against this branch and that PRs must also be in 4.2